### PR TITLE
Help suffix space

### PR DESCRIPTION
**Pull request by zenlint**
Normalize the log for docker subcommand usage, delete the trailing space at the end of option.
Add checking for for suffix of option in each line of help, to make sure no extra space in it.

### CODE REVIEW

File: integration-cli/docker_cli_help_test.go [hunk 1]
```diff
@@ -130,6 +130,11 @@ func TestHelpTextVerify(t *testing.T) {
 				if strings.HasPrefix(line, "  -") && strings.HasSuffix(line, ".") {
 					t.Fatalf("Help for %q should not end with a period: %s", cmd, line)
 				}
+
+				// Options should not end with a space
+				if strings.HasSuffix(line, " ") {
```
Comment (duglin) [inline integration-cli/docker_cli_help_test.go]: Can you please add this check for all lines of the help output? Not just the options but the main help too?  Back near line 65

Comment (zenlinTechnofreak) [inline integration-cli/docker_cli_help_test.go]: @duglin  done it. Thx

**Comment by duglin** [comment]
hmm, it looks like you moved it, I wanted it in both places  :-)

**Comment by zenlinTechnofreak** [comment]
@duglin sorry for careless, remodified it again.

**Comment by duglin** [comment]
@zenlinTechnofreak thanks
LGTM

/cc @tiborvass

**Comment by tiborvass** [comment]
LGTM

**Pull request merged**